pyproject.toml

```
@@ -0,0 +1,38 @@
+[tool.poetry]
+name = "affinity-groups-slackbot"
+version = "0.1.0"
+description = ""
+authors = ["Your Name <you@example.com>"]
+readme = "README.md"
+packages = [{include = "affinity_groups_slackbot"}]
+
+[tool.poetry.dependencies]
+python = "^3.11"
+aiodns = "3.0.0"
+aiohttp = "3.8.4"
+async-timeout = "4.0.2"
+attrs = "22.2.0"
+cffi = "1.15.1"
+chardet = "5.1.0"
+click = "8.1.3"
+idna = "2.8"
+jinja2 = "3.1.2"
+markupsafe = "2.1.2"
+multidict = "4.5.2"
+pycares = "4.3.0"
+pycparser = "2.19"
+slackclient = "2.9.4"
+werkzeug = "2.2.3"
+yarl = "1.8.2"
+itsdangerous = "^2.1.2"
+flask = "^2.2.3"
+
+
```

```suggestion
slackclient = "2.9.4"
flask = "^2.2.3"

```
This is enough for poetry to do the rest